webpack/src/utils.ts

```diff
@@ -105,7 +106,10 @@ export function getWebpackConfig () {
       resolve: {
         extensions: ['.json', '.js', '.jsx', '.css']
       },
-      devtool: 'source-map'
+      devtool: 'source-map',
+      devServer: {
+        publicPath: path.join('/${bundlePath}/')
```

Comment:
@jeremyrajan - great question.
The [webpack.js.org](https://webpack.js.org) documentation explicitly states that `devServer.publicPath` **must** start and end with a `/`.

Ref: https://webpack.js.org/configuration/dev-server/#devserver-publicpath-